Changelog.md

```diff
@@ -4,6 +4,10 @@
 
 Nothing here yet.
 
+## v17.1.0 (2023-01-02)
```

Comment:
please, remove this "release number" here, you'd rather add the changelog entry to the current (master) section.

FYI, it's in the Github PR template, as well as in the "How to Contribute" documentation: https://github.com/workalendar/workalendar/blob/master/docs/contributing.md#the-final-steps